source/extensions/transport_sockets/tls/cert_validator/spiffe/spiffe_validator.h

```diff
@@ -30,12 +32,26 @@ namespace Tls {
 
 using X509StorePtr = CSmartPtr<X509_STORE, X509_STORE_free>;
 
-class SPIFFEValidator : public CertValidator {
+struct SpiffeData {
+  absl::flat_hash_map<std::string, CSmartPtr<X509_STORE, X509_STORE_free>> trust_bundle_stores;
+  std::vector<bssl::UniquePtr<X509>> ca_certs;
+
+  int64_t spiffe_refresh_hint;
+  int64_t spiffe_sequence;
```

Comment:
nit for style: may end with `_` according to our style.

Please initialize the int64_t with in-class member initializers.